providers/apis/dydx/parse.go

```diff
@@ -180,35 +185,118 @@ func (h *APIHandler) ConvertExchangeConfigJSON(
 			normalizeByPair = &temp
 		}
 
+		denom, err := ConvertDenomByProvider(cfg.Ticker, exchange)
+		if err != nil {
+			return nil, fmt.Errorf("failed to convert denom by provider: %w", err)
+		}
+
+		metaData, err := ExtractMetadataFromTicker(cfg.Ticker, exchange)
+		if err != nil {
+			return nil, fmt.Errorf("failed to extract metadata from ticker: %w", err)
+		}
+
 		// Convert to a provider config.
 		providers = append(providers, mmtypes.ProviderConfig{
 			Name:            exchange,
-			OffChainTicker:  ConvertDenomByProvider(cfg.Ticker, exchange), // Convert the ticker to the provider's format.
+			OffChainTicker:  denom, // Convert the ticker to the provider's format.
 			Invert:          cfg.Invert,
 			NormalizeByPair: normalizeByPair,
+			Metadata_JSON:   metaData,
 		})
 	}
 
 	return providers, nil
 }
 
+// ExtractMetadataFromTicker extracts json-metadata from a ticker, based on the exchange. Specifically,
```

Comment:
This is likely not going to be raydium specific, could we move these docs to raydium